File: test/fixtures/php-guzzle/get_user_agent.php
<?php
require 'vendor/autoload.php';

use GuzzleHttp\Client;

$client = new Client();

$response = $client->get('http://localhost:28139/vc/moviesmagic', [
    'query' => [
        'p' => '5',
        'pub' => 'testmovie',
        'tkn' => '817263812'
    ],
    'headers' => [
        'x-msisdn'   => 'XXXXXXXXXXXXX',
        'user-agent' => 'Mozilla Android6.1'
    ]
]);

File: test/fixtures/java-httpurlconnection/get_user_agent.java
import java.io.IOException;
import java.io.InputStream;
import java.net.HttpURLConnection;
import java.net.URL;
import java.util.Scanner;

class Main {

    public static void main(String[] args) throws IOException {
        URL url = new URL("http://localhost:28139/vc/moviesmagic?p=5&pub=testmovie&tkn=817263812");
        HttpURLConnection httpConn = (HttpURLConnection) url.openConnection();
        httpConn.setRequestMethod("GET");

        httpConn.setRequestProperty("x-msisdn", "XXXXXXXXXXXXX");
        httpConn.setRequestProperty("user-agent", "Mozilla Android6.1");

        InputStream responseStream = httpConn.getResponseCode() / 100 == 2
                ? httpConn.getInputStream()
                : httpConn.getErrorStream();
        Scanner s = new Scanner(responseStream).useDelimiter("\\A");
        String response = s.hasNext() ? s.next() : "";
        System.out.println(response);
    }
}
